src/clp_bench/clpg_executor.py

```
@@ -2,9 +2,9 @@
 import subprocess
 import time
 
-from .executor import BenchmarkingMode, BenchmarkingSystemMetric, CPTExecutorBase
+from .executor import BenchmarkingMode, BenchmarkingSystemMetric, CPTExecutorBase, BenchmarkingResult
```

Ruff

Line too long (101 > 100)

(E501)